### PR TITLE
Inserted J2objc link

### PR DESCRIPTION
**Pull request by confile**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # j2objc-gradle
-Gradle plugin for j2objc (Java to Objective-C transpiler)
+Gradle plugin for [j2objc](https://github.com/google/j2objc) (Java to Objective-C transpiler)
```
Comment (brunobowden) [inline README.md]: Change to J2ObjC... I've noticed them standardizing on that capitalization.

**Comment by brunobowden** [comment]
@Confile - also please squash the commit when you do this, see CONTRIBUTING.md for details

File: README.md (hunk 1 above)
Comment (brunobowden) [inline README.md]: @confile - can you capitalize it to J2ObjC? Thanks.

**Comment by brunobowden** [comment]
@Confile - please squash the commits so that it's a single commit rather than two. There's a section in the CONTRIBUTING file about how to do this:

https://github.com/brunobowden/j2objc-gradle/blob/master/CONTRIBUTING.md#rebasing-and-merging

**Comment by advayDev1** [comment]
Please start at the Preparation heading on that file though.

**Comment by brunobowden** [comment]
Preparation:
https://github.com/brunobowden/j2objc-gradle/blob/master/CONTRIBUTING.md#preparation

**Comment by confile** [comment]
@brunobowden Done

**Pull request merged**